### ISSUE TITLE
Found a working PlayPlay token!

### ISSUE THREAD

**Issue by PwLDev** [cycyrild/SpotifyDL#4]
I've found a working playplay request token, it always returns 200 when making requests to it (works with both v2 and v3)

Hex Token: `01a7cbe0d515351f69c2abf73b337a6b` [Source](https://github.com/DaXcess/node-spotify-ap/blob/81e66fd5122936f299118fd75fd47420076f12e8/lib/audio/AudioKeyManager.ts#L84)

However, as far as I researched there is no playplay deobfuscation algorithm that uses this token.

**Comment by cycyrild** [comment]
Spotify is very responsive to this and revokes the tokens immediately. The only "stable" method is to use Widevine with a valid dump (real device, android simulators don't work).

**Comment by SuisChan** [comment]
> However, as far as I researched there is no playplay deobfuscation algorithm that uses this token.

Exactly, one should think of it as a pair of public and private keys, where the token is the public key for requesting a key, and the decryptor is... obviously the private key. And each part without the other is simply useless...

**Comment by PwLDev** [comment]
Unfortunately I don't have more info about the decryptor (playplayIntent) key, as I only tested that public request token from the repo I referenced, and still as of today, is the only one that worked for me returning a valid 200 response.
